README.md

```diff
@@ -30,40 +30,47 @@ for the next steps.
 
 ### Setup Environment
 
-1. Create a `.env` file in the root of your project and add to it the following contents
+1. Copy over the `.env.local` file into a `.env` file and fill out the `SHOPIFY_API_KEY` and `SHOPIFY_API_SECRET` fields
 ```
-SHOPIFY_API_KEY=[your api key]
-SHOPIFY_API_SECRET=[your api secret]
+cp .env.local .env
 ```
```

Comment:
I'm a fan of making `DJANGO_SECRET` a default key which is filled in by step 2, but I'm not sure a new file is needed. Totally a personal opinion though

Reply:
yeah it's basically the same as copying from the readme, just like half a second faster 😆  should I remove it? It's fine either way